pkg/sass-types/README.md

```diff
@@ -0,0 +1,6 @@
+# sass-types
+
+This package is a dual-publish of the [Sass JS API](https://sass-lang.com/documentation/js-api/)
+types. The `sass` and `sass-embedded` packages **still contain types**. This package is for
+users who _only_ need the Sass JS types (such as for `@types/gulp-sass`). It is published
```

Comment:
`@types/gulp-sass` does not use this _yet_, but I will use it once published to implement the type declarations for gulp-sass 6.x which uses the modern JS API (types for gulp-sass 5.x are implemented with a dependency on `@types/node-sass`, which was not supporting all the recent options even for the legacy API, but was at least a type-only package not forcing to install a different sass implementation than the one intended to be used)